### ISSUE TITLE
[Accessibility] PrimaryButton has the same style as DefaultButton in High Contrast Black/White mode

### ISSUE THREAD

**Issue by stephenzsy** [OfficeDev/office-ui-fabric-react#1751]
### Bug Report

- __Package version(s)__: Fabric Core >=5.1.0-0 <6.0.0-0 and Fabric React >=2.23.0 <3.0.0
- __Browser and OS versions__: Microsoft Edge 40.15063.0.0 on Windows 10 1703 15063.250

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? No

Requested priority: High

Products/sites affected: (if applicable)

#### Describe the issue:

In High Contrast Mode in Microsoft Edge, PrimaryButton has the same color style as DefaultButton

#### Actual behavior:

In High Contrast Mode in Microsoft Edge, PrimaryButton has the same color style as DefaultButton

#### Expected behavior:

In High Contrast Mode in Microsoft Edge, PrimaryButton should have different color styles as DefaultButton

**Comment by mdahamiwal** [comment]
Valid concern. @betrue-final-final, is there a way to align it with normal mode? 

**Normal** 
![image](https://cloud.githubusercontent.com/assets/12109362/25836457/91a92a9c-34a4-11e7-8ddc-aac832f13da2.png)

**High contrast**
![image](https://cloud.githubusercontent.com/assets/12109362/25836469/ab8c241e-34a4-11e7-87a8-016af18f0e2d.png)

**Comment by betrue-final-final** [comment]
Yes, I would use "selected background" color (cyan) for the border. 
![image](https://cloud.githubusercontent.com/assets/20363037/25857988/7af235ec-348f-11e7-9b64-5abd42b2ec39.png)

**Comment by mdahamiwal** [comment]
@betrue-final-final isn't cyan border or fill supposed to be focus or hover indicator? I am not very convinced with current focus and hover treatment. 

![primary btn-focus](https://cloud.githubusercontent.com/assets/12109362/25910783/353a8428-35cf-11e7-9c0f-1e563577d0fe.gif)

**Comment by mdahamiwal** [comment]
I like the hover and focus treatment provided by windows using background color:

![gif-windows-button](https://cloud.githubusercontent.com/assets/12109362/25911035/0d423ef6-35d0-11e7-8013-9c21607c8aa8.gif)

what you think ?

**Comment by betrue-final-final** [comment]
Yeah, this problem is pretty deep. No hover, focus is unevenly presented...for one, we can make hover cyan, which is what is done all over Windows, and even in OneDrive. Let me get back to you with a proposal after I find out from the Windows designers how they handle accent buttons. I'm having a hard time finding an accent button in Windows.

![image](https://cloud.githubusercontent.com/assets/20363037/25915159/6c60ff90-3575-11e7-9ff9-cf82b223dfbb.png)
